ironfist/src/cpp/adventure/adv.cpp

```diff
@@ -2,6 +2,62 @@
 #include "adventure/map.h"
 #include "game/game.h"
 #include "scripting/hook.h"
+#include "prefs.h"
+
+extern void __fastcall NormalDialog(char *, int, int, int, int, int, int, int, int, int);
+
+const int BUTTON_CODE_OKAY = 30725;
+const int BUTTON_CODE_CANCEL = 30726;
+
+extern long __fastcall KBTickCount();
+
+int advManager::ProcessDeSelect(struct tag_message *GUIMessage_evt, int *a3, class mapCell * *a4) {

```

Comment:
Rename a3 and a4 to meaningful variable names. I don't have strong feelings, but we've just been using "tag_message*" or "mapCell*" instead of e.g: "class mapCell*" for parameter declarations elsewhere.